ruby/vampires.rb

```diff
@@ -0,0 +1,52 @@
+puts "How many employees are you processing?"
+num_employees = gets.chomp.to_i
```

Comment:
Great use of method chaining, one thing you could up here is the naming. While num_employees makes sense, something like total_employees or employees_count may be more explicit